cmd/generate/main_test.go

```diff
@@ -63,3 +65,46 @@ metadata:
   name: test-config-map
 `, buf.String())
 }
+
+func TestGenerateCmd(t *testing.T) {
+	c := cmd.NewRootCommand()
+	c.AddCommand(NewGenerateCommand())
+
+	cr := `
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simplest
+spec:
+  images:
+    tempo: docker.io/grafana/tempo:x.y.z
+    tempoQuery: docker.io/grafana/tempo-query:x.y.z
+    tempoGateway: quay.io/observatorium/api
+  storage:
+    secret: minio-test
+  storageSize: 1Gi
+`
+	c.SetIn(strings.NewReader(cr))
+
+	out := &strings.Builder{}
+	c.SetOut(out)
+	c.SetErr(out)
+
+	c.SetArgs([]string{"generate"})
+	_, err := c.ExecuteC()
+	require.NoError(t, err)
+
+	require.Contains(t, out.String(), `
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
```

Comment:
Maybe: should we fix this in the future?  A `null` timestamp sounds weird for me

Reply:
Done